.travis.yml

```diff
@@ -1,6 +1,19 @@
 sudo: false
-install: ./autogen.sh
 language: c
 compiler:
     - clang
     - gcc
+os:
+    - linux
+    - osx
+matrix:
+  exclude:
+    - os: osx
+      compiler: gcc
+  fast_finish: true
+
+before_install:
+    - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew update; fi
+install: ./autogen.sh
+script: ./configure --prefix=${TRAVIS_BUILD_DIR}/install_dir &&
```

Comment:
For fabtests, I used $HOME, which IMO seems slightly better, since we'll want to install multiple libraries in order to better test the build against all providers.